From dc0e2edc5bfc9db15dce9869198099131f64f42d Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Thu, 24 Mar 2016 16:48:46 -0700
Subject: [PATCH 2/2] Avoid stray As next to IDLW icons

* lisp/progmodes/idlw-toolbar.el (idlwave-toolbar-add-everywhere):
Use "" for empty labels, not "a", as the latter now displays stray
"A"s (Bug#18997).
---
 lisp/progmodes/idlw-toolbar.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lisp/progmodes/idlw-toolbar.el b/lisp/progmodes/idlw-toolbar.el
index 0e0714e..d857bfd 100644
--- a/lisp/progmodes/idlw-toolbar.el
+++ b/lisp/progmodes/idlw-toolbar.el
@@ -439,7 +439,7 @@ idlwave-toolbar-cont-icon
 \"                            \",
 \"                            \",
 \"                            \"};")
-  "The Cont icon.")	  
+  "The Cont icon.")
 
 (defvar idlwave-toolbar-to-here-icon
   (idlwave-toolbar-make-button
@@ -918,7 +918,7 @@ idlwave-toolbar-add-everywhere
 		     (help (aref x 3))
 		     (key (vector 'tool-bar func))
 		     (def (list 'menu-item
-				"a"
+				""
 				func
 				:image (symbol-value icon)
 				:visible show
-- 
2.5.5

